packages/taquito/src/constants.ts

```diff
@@ -88,19 +109,37 @@ export enum ChainIds {
   OXFORDNET2 = 'NetXxWsskGahzQB',
 }
 
+// A fixed fee reveal operation gasLimit accepted by both simulate and injection endpoint is between 1.2-5 times of actual gas consumption
 export const getRevealGasLimit = (address: string) =>
-  Math.round((getRevealGasLimitInternal(address) * 11) / 10);
+  Math.round((getRevealGasLimitInternal(address) * 35) / 10);
 
 const getRevealGasLimitInternal = (address: string) => {
   switch (address.substring(0, 3)) {
     case 'tz1':
-      return DEFAULT_GAS_LIMIT.REVEAL_TZ1;
+      return REVEAL_GAS_LIMIT.TZ1;
+    case 'tz2':
+      return REVEAL_GAS_LIMIT.TZ2;
+    case 'tz3':
+      return REVEAL_GAS_LIMIT.TZ3;
+    case 'tz4':
+      return REVEAL_GAS_LIMIT.TZ4;
+    default:
+      throw new Error(`Cannot estimate reveal gas limit for ${address}`);
+  }
+};
+
+export const getRevealFee = (address: string) => getRevealFeeInternal(address);
```

Comment:
what's the reason that getRevealFee is layered on top of getRevealFeeInternal?

Reply:
good call out, was trying to have same structure as `getRevealGasLimit` but getRevealFee doesn't add logic value will update